docs/DeveloperGuide.md

```diff
@@ -300,6 +300,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `TutorAid` and the **Actor** is the `tutor`, unless specified otherwise)
 
+**Use Case 1: Add a student**
+
+**MSS**
+
+1. Tutor collates information related to the student to be added
+
+2. Tutor adds the student by passing the student's name and other details to TutorAid
+
+3. TutorAid adds the student and displays a message indicating that this has been done
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. Tutor enters the `add` command without using the correct format
+
+  * 1a1. TutorAid displays an error message and requests the tutor to re-enter the command
+
+  * 1a2. Tutor re-enters the command, along with the necessary arguments
+
+    Steps 1a1-1a2 are repeated until the data entered is correct.
+
+    Use case resumes from step 3.
+
+* 3a. Tutor realises that they added the details wrongly
```

Comment:
Should this extension be numbered 2a instead?

Reply:
Agree, I think need to standardise the numbering here but otherwise LGTM

Reply:
Fixed, thanks!